tools/deploy-contract.ts

```diff
@@ -18,26 +18,33 @@ import { arweave, initialize, warp } from './utilities';
   // load state of contract
   const ARNS_CONTRACT_TX_ID =
     process.env.ARNS_CONTRACT_TX_ID ??
-    'E-pRI1bokGWQBqHnbut9rsHSt9Ypbldos3bAtwg4JMc';
+    'blAgYxAdX2Ry-nt6aH2ixgvJXbpsEYm28NgJgyqfs-U';
```

Comment:
just reverting this to testnet to avoid confusion with other devs using these tool scripts